pygsti/report/workspaceplots.py

```diff
@@ -662,42 +697,82 @@ def _circuit_color_scatterplot(circuit_structure, sub_mxs, colormap,
     plotly.Figure
     """
     g = circuit_structure
-    xvals = g.used_xs
-    yvals = g.used_ys
+    
+    if isinstance(g, _PlaquetteGridCircuitStructure):
+        xvals = g.used_xs
+        yvals = g.used_ys
 
     if addl_hover_submxs is None:
         addl_hover_submxs = {}
 
     if hover_info:
-        hover_info = _create_hover_info_fn(circuit_structure, xvals, yvals, sum_up, addl_hover_submxs)
-
+        if isinstance(g, _PlaquetteGridCircuitStructure):
+            hover_info = _create_hover_info_fn(circuit_structure, xvals, yvals, sum_up, addl_hover_submxs)
```

Comment:
If we use `g.used_xs` and `g.used_ys` here, we could probably remove the `xvals`/`yvals` definition above. Minor thing, but probably helpful to developers to not have those variables defined when thinking about the CircuitList codepath instead of the PlaqGrid one.

Reply:
Good catch, I hadn't noticed when updating this that these variables are only used in one place in this function, and that is already wrapped in an isinstance check. Updated.